src/wp-includes/option.php

```diff
@@ -643,50 +643,150 @@ function wp_load_alloptions( $force_cache = false ) {
 }
 
 /**
- * Loads and primes caches of certain often requested network options if is_multisite().
+ * Primes specific network options for the current network into the cache with a single database query.
  *
- * @since 3.0.0
- * @since 6.3.0 Also prime caches for network options when persistent object cache is enabled.
+ * Only network options that do not already exist in cache will be loaded.
+ *
+ * If site is not multisite, then call wp_prime_option_caches().
+ *
+ * @since 6.6.0
+ *
+ * @see wp_prime_network_option_caches()
+ *
+ * @param string[] $options An array of option names to be loaded.
+ */
+function wp_prime_site_option_caches( array $options = array() ) {
```

Comment:
```suggestion
function wp_prime_site_option_caches( array $options ) {
```

Similar to `wp_prime_network_option_caches` ?